ref_code_gen.py

```diff
@@ -84,6 +93,7 @@ def check_digit(cls, candidate: str,
         crc = crc32(base.encode('utf-8'))
         digit = crc % 10
         out = f'{base}{digit}'
+        # print(candidate, base_len, base, crc, digit, out)
```

Comment:
lets avoid adding commented out code

Reply:
I have intended to keep that as a print statement helps to understand the code, when someone revisits the code.